src/bitsymuse.js

```diff
@@ -0,0 +1,190 @@
+/**
+ðŸšª
+@file Bitsymuse
+@summary A variety of Bitsy sound and music handlers
+@license MIT
+@version 1.0.0
+@requires 4.8, 4.9
+@author David Mowatt
+
+@description
+A hack that a variety of audio controlls, including music that changes as you move between rooms.
+If the same song is played as you move between rooms, the audio file will continue playing.
+
+HOW TO USE:
+1. Place your audio files somewhere relative to your bitsy html file (in the zip if you're uploading to itch.io)
+2. Copy-paste `<audio id="<sound ID>" src="<relative path to sound file>"></audio>` into the <head> of your document.
+   You need to do it once for each sound file you are adding, and each needs a unique sound ID. Add "loop" after the src=""
+   tag if it's music tha's going to loop (so `<audio id ="<sound id> src="<path>" loop></audio>
+3. See var hackOptions below to set up the TRACK LIST for rooms you move through.
+4. Copy-paste this script into a script tag after the bitsy source.
+
+In addition to the track list, that will play audio based on the room number, you have access to the following
+commands you can add to dialogue:
+
+1. soundeffect ("<sound ID>") will play a sound without interrupting the music
+2. music ("<sound ID>?) will change the music as soon as it is called in the dialogue
+3. musicEnd ("<sound ID>") will change the music once the dialogue box closes
+
+You can call both music and musicEnd in the same dialogue, to e.g. change the music while you speak to a character
+and then restart the regular room music once you stop speaking to them. "S" can be used as a sound ID for music
+and musicEnd to Silence the music.
+
+*/
+
+'use strict';
+import bitsy from "bitsy";
+import {
+	before,
+	after,
+	inject
+} from "./kitsy-script-toolkit.js";
+
+var hackOptions = {
+    MusicByRoom: {
```

Comment:
tabs instead of spaces, and the contents of `MusicByRoom` should be tabbed further